doc/cephfs/fs-volumes.rst

```diff
@@ -312,6 +312,7 @@ The output format is json and contains fields as follows.
 * created_at: time of creation of snapshot in the format "YYYY-MM-DD HH:MM:SS:ffffff"
 * data_pool: data pool the snapshot belongs to
 * has_pending_clones: "yes" if snapshot clone is in progress otherwise "no"
+* pending_clones: list of in progress or pending clones if exist otherwise this field will not exist
```

Comment:
s/this field will not exist/this field is not shown ?
Adding a sample output helps here.